### PR TITLE
Fix status mapping

### PR DESCRIPTION
**Pull request by luthfib**
Display status will now show the correct display status. Add tests to confirm a bunch of cases

### CODE REVIEW

File: src/fhir/models/condition.ts [hunk 1]
```diff
@@ -277,31 +277,55 @@ export class ConditionModel extends FHIRModel<fhir4.Condition> {
         case "resolved":
           return "Inactive";
         default:
-          return "Unknown";
+          return "";
       }
     }
 
+    // What to show if patient record resource.
```
Comment (pmadria3) [inline src/fhir/models/condition.ts:284]: Not sure what this comment means

File: src/fhir/models/condition.ts [hunk 2]
```diff
@@ -277,31 +277,55 @@ export class ConditionModel extends FHIRModel<fhir4.Condition> {
...
+    function byVerificationStatus(code: VerificationStatus | undefined) {
```
Comment (pmadria3) [inline src/fhir/models/condition.ts:285]: rename function?

Comment (pmadria3) [reply src/fhir/models/condition.ts:285]: maybe whichVerificationStatus? same with the clinicalStatus?

File: src/fhir/models/condition.ts (hunk 1 above)
Comment (luthfib) [reply src/fhir/models/condition.ts:284]: Like if the record is not other provider records but patient owned resource

**Review by pmadria3 — APPROVED** [review]
Couple nits - otherwise LGTM

File: src/fhir/models/condition.ts (hunk 1 above)
Comment (luthfib) [reply src/fhir/models/condition.ts:284]: I moved it

Comment (pmadria3) [reply src/fhir/models/condition.ts]: Gotcha - wasn't sure what the comment meant but makes sense

**Review by pmadria3 — APPROVED** [review]

**Review by mrosata — APPROVED** [review]

**Pull request merged**